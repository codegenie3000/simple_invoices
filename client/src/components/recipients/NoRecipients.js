import React from 'react';

export default props => {
    return (
        <div>You don't have any recipients. Please add one.</div>
    );
}