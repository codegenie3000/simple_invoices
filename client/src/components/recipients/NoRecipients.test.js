import React from 'react';
import { render, mount, shallow } from 'enzyme';

import NoRecipients from './NoRecipients';

describe('No Recipients Component', () => {
    it('should render correctly', ()=> {
        const component = render(
            <NoRecipients/>
        );
        expect(component).toMatchSnapshot();
    });
});