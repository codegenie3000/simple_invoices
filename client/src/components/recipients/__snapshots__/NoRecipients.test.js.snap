// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No Recipients Component should render correctly 1`] = `
<div>
  You don't have any recipients. Please add one.
</div>
`;
